_config.yml

```diff
@@ -60,6 +60,36 @@ lc_post_survey: "https://www.surveymonkey.com/r/lcpostworkshopsurvey?workshop_id
 instructor_pre_survey: "https://www.surveymonkey.com/r/instructor_training_pre_survey?workshop_id="
 instructor_post_survey: "https://www.surveymonkey.com/r/instructor_training_post_survey?workshop_id="
 
+# Workshop specific values
+#########################################
+# 
+
+host_name: "Jureca"
+#host_login: "kay.ichec.ie"
+host_location: "JSC, Julich Supercomputing Centre"
+#host_homedir: "/ichec/home/course01"  # watch out due to course accounts
+host_prompt: "[user@laptop ~]$"
+sched_name: "SBATCH" # Check
+sched_submit: "sbatch"
+sched_submit_options: "-A course01" # check
+sched_flag_time: "--time "
+sched_flag_node: "--nodes "
+sched_status: "sstat"
+sched_flag_user: "-u yourUsername"
+sched_del: "scancel"
+sched_interactive: "sbatch"
+sched_comment: "#SBATCH"
+sched_flag_name: "-N" # EDITME
+sched_hist: "" # EDITME
+sched_flag_histdetail: "" # EDITME 
+module_load: "module load LAMMPS/9Jan2020-cuda intel-para/2019a Stages/Devel-2019a"
+module_usage: "module use /usr/local/software/jureca/OtherStages"
+run_openmp: "srun"
+run_mpi: ""
```

Comment:
You do need `srun` here though, but srun on its own is probably not enough. For mpirun you would also need `mpirun -n {number of cores}`, though I am not sure how we would handle this.

Reply:
if it is left as `mpirun -n` and then we can have a generic core count (eg 4)